From 168ba7a681be73ac024438e33e14fde1d5aea97d Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 30 Mar 2018 10:02:24 +0800
Subject: [PATCH 1/2] tic hang

Upstream-Status: Inappropriate [configuration]

'tic' of some linux distributions (e.g. fedora 11) hang in an infinite
loop when processing the original file.

Signed-off-by: anonymous

Rebase to 6.1
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 misc/terminfo.src | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/misc/terminfo.src b/misc/terminfo.src
index 84f4810..6b385ec 100644
--- a/misc/terminfo.src
+++ b/misc/terminfo.src
@@ -5562,12 +5562,11 @@ konsole-xf3x|KDE console window with keyboard for XFree86 3.x xterm,
 # The value for kbs (see konsole-vt100) reflects local customization rather
 # than the settings used for XFree86 xterm.
 konsole-xf4x|KDE console window with keyboard for XFree86 4.x xterm,
-	kend=\EOF, khome=\EOH, use=konsole+pcfkeys,
-	use=konsole-vt100,
-
-konsole+pcfkeys|konsole subset of xterm+pcfkeys,
-	kcbt=\E[Z, use=xterm+pcc2, use=xterm+pcf0,
-	use=xterm+pce2,
+	kend=\EOF, kf1=\EOP, kf13=\EO2P, kf14=\EO2Q, kf15=\EO2R,
+	kf16=\EO2S, kf17=\E[15;2~, kf18=\E[17;2~, kf19=\E[18;2~,
+	kf2=\EOQ, kf20=\E[19;2~, kf21=\E[20;2~, kf22=\E[21;2~,
+	kf23=\E[23;2~, kf24=\E[24;2~, kf3=\EOR, kf4=\EOS,
+	khome=\EOH, use=konsole-vt100,
 
 # Obsolete: vt100.keymap
 # KDE's "vt100" keyboard has no relationship to any terminal that DEC made, but
-- 
1.8.3.1

